test/parallel/test-npm-install.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const common = require('../common');
+
+const path = require('path');
+const spawn = require('child_process').spawn;
+const assert = require('assert');
+
+common.refreshTmpDir();
+
+const npmPath = path.join(
+  common.testDir,
+  '..',
+  'deps',
+  'npm',
+  'bin',
+  'npm-cli.js'
+);
```

Comment:
less likely to break if npm changes their entry point. prob doesnt matter though :)